.golangci.yml

```diff
@@ -0,0 +1,4 @@
+run:
```

Comment:
@rikatz What linters should be added here?

Reply:
i would say to match at least the one we have in the k/release https://github.com/kubernetes/release/blob/master/.golangci.yml